### PR TITLE
Mental health & Spiritual Health Sections Enhancement (Solved)

### PR DESCRIPTION
**Pull request by debuuuu007**
Still I have some doubts on which font will look better. I have almost used 30 fonts but the default one looks more attractive in terms of reading. Also added background images on "mental health" and "spiritual wellness" section, if its not looking good do let me know I will change it with something else. Till now I think I have done my work.

Any type of suggestions are always welcomed, Thank you.

### CODE REVIEW

File: css_files/mental_health.css
Comment (ritika728) [inline css_files/mental_health.css:1]: As per my understanding, the code addition for footer, social icons & media queries for navbar is not being utilized anywhere. You can remove these to increase readability.

Comment (ritika728) [reply css_files/mental_health.css:1]: Same goes with the Spiritual wellness CSS file

**Comment by debuuuu007** [comment]
Ouk mam I have just copy pasted the social media as these were already written in a pre build one. If u want i will remove them for sure. 

And sorry i have added that image by mistake.

**Comment by ritika728** [comment]
No issues, looks good to merge 👍
Thank you for your contribution!

**Comment by debuuuu007** [comment]
Thanks mam, this was my very first contribution in Open Source. Hope you like my work. Also looking forward to contribute more in the upcoming days.

And mam how we can see our scores ? Is there any system to see score for now or this will automatically be keeps on updating ?

**Comment by ritika728** [comment]
Yes, you definitely did a good job 🚀
You would be able to see your scores in the leaderboard, which would be revealed after a few days. Your score will automatically update with every successfully merged PR with relevant labels.

**Comment by debuuuu007** [comment]
Mam anything went wrong?

You have assigned me again in this issue?

**Comment by ritika728** [comment]
No, nothing went wrong. I've assigned it for the leaderboard purpose.

**Pull request merged**